### PR TITLE
Issue 338 - Fixes cdn error and uses npm for less compiling

### PR DESCRIPTION
**Pull request by vertein**
I also ran into https://github.com/UW-Madison-DoIT/uw-frame/issues/338 about 70% of the time when I was on wireless, but almost 0% of the time when I was wired.

This also takes the idea of compiling all the less in one spot from : https://github.com/UW-Madison-DoIT/uw-frame/pull/340/ while keeping the version numbering that we are currently using.

### CODE REVIEW

**Comment by vertein** [comment]
Failing travis ci builds also gave me pause.  There were two issues.  1. fixed maven pom to point at directory where package.json was.  2. Fixed a cp error in build.js where if directory already existed, the build would failed.  A couple of versions ago we were removing that directory explicitly before trying to copy over.  This goes back to that.

File: package.json [hunk 1]
```diff
@@ -2,13 +2,17 @@
   "name": "uw-frame",
   "version": "2.9.0",
   "description": "MyUW's frame project for creating angular based microservices.",
+  "config" : {
+    "javaversion" : "1"
```
Comment (apetro) [inline package.json]: What's `javaversion`? Under what circumstances ought it to be incremented? How would I know that?

**Comment by paulerickson** [comment]
npm runs maven runs npm... I think that a circular dependency like this makes the project more difficult to understand and maintain

**Comment by vertein** [comment]
@paulerickson I do too.  I was/am actually fine with maven compiling on their own less (https://github.com/UW-Madison-DoIT/uw-frame/pull/340#issuecomment-255415981).

I'd like to look into removing a lot of the npm calling maven things, like the clean scripts and the build scripts in the future.

**Pull request merged**